recipe/test_mpi4py.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+import mpi4py
+from mpi4py import MPI
+import mpi4py.futures
+
+# on Windows this checks MSMPI_VER
+print(MPI.get_vendor())
+
+print(MPI.Get_library_version())
+
+# verify get_config paths exist (not links to build env)
+config = mpi4py.get_config()
+print(f"config={config}")
+for name in ("mpicc", "mpicxx", "mpifort"):
+    if name in config:
+        path = config[name]
+        assert os.path.exists(path), f"{name} missing: {path}"
+    else:
+        raise(Exception(f"Missing {name}"))
```

Comment:
```suggestion
for name in ("mpicc", "mpicxx", "mpifort"):
    assert name in config
    path = config[name]
    assert os.path.exists(path), f"{name} missing: {path}"
```

This should work, too, and is more compact.